test/aktivitetskrav/VurderAktivitetskravTest.tsx

```diff
@@ -135,14 +134,10 @@ describe("VurderAktivitetskrav", () => {
       changeTextInput(beskrivelseInput, tooLongBeskrivelse);
       clickButton("Lagre");
 
-      expect(screen.getByText("Vennligst angi årsak")).to.exist;
-      expect(
-        screen.getByText(
-          maxLengthErrorMessage(vurderAktivitetskravBeskrivelseMaxLength)
-        )
-      ).to.exist;
+      expect(await screen.findByText("Vennligst angi årsak")).to.exist;
+      expect(screen.getByText("1 tegn for mye")).to.exist;
```

Comment:
Hvorfor én `findBy` og én `getBy`? 🤔

Reply:
Når man trykker Lagre så skjer valideringen async så derfor måtte jeg endre til `findby` for å teste at valideringsfeilen vises. Siden vi har ventet på at den ene valideringsfeilen vises så trenger vi ikke vente på den andre. Men kan godt bruke `findBy `på den også selv om det strengt tatt ikke trengs. Så slipper man problemer hvis man skulle endre rekkefølge på disse f.eks.